index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 var execall = require('execall');
-var re = /^\s*[^$]\s*(?:-([a-z-]), )?--([a-z-]+) +(.*)$/igm;
+var re = /^\s*[^$]\s*(?:-([a-z-]),\s+)?--([a-z-]+) +(.*)$/igm;
```

Comment:
oh, yeah, didn't thought about that, because of the `m` flag right ?

Is there another way ?